shell/common/skia_util.cc

```diff
@@ -155,6 +147,17 @@ bool PopulateImageSkiaRepsFromPath(gfx::ImageSkia* image,
   return succeed;
 }
 
+#if defined(OS_WIN)
+bool ReadImageSkiaFromICO(gfx::ImageSkia* image, HICON icon) {
+  // Convert the icon from the Windows specific HICON to gfx::ImageSkia.
+  SkBitmap bitmap = IconUtil::CreateSkBitmapFromHICON(icon);
+  if (bitmap.isNull())
+    return false;
+
+  image->AddRepresentation(gfx::ImageSkiaRep(bitmap, 1.0f));
+  return true;
+#endif
+
 }  // namespace util
 
-}  // namespace electron
+}  // namespace util
```

Comment:
i don't think this should have been changed?

Reply:
Has already been fixed in a later commit (ab9e458)! Problem was I forgot to paste the closing bracket in the Windows file, and the linter auto-changed these names because the brackets were wonky. :)